src/index.js

```diff
@@ -125,7 +135,7 @@ class Game extends React.Component {
 
 // ========================================
 
-ReactDOM.render(<Game />, document.getElementById("root"));
+ReactDOM.render(<Game/>, document.getElementById("root"));
```

Comment:
previos commit add space this one remove, maybe just do it in one commit without change